guides/v2.3/graphql/queries/store-config.md

```diff
@@ -339,3 +225,118 @@ The [ProductInterface]({{ page.baseurl }}/graphql/reference/products.html#Produc
   }
 }
 ```
+
+### Extend configuration data
+
+You can add your own configuration to the `storeConfig` query within your own module.
+
+To do this, configure the constructor argument `extendedConfigData` in the `argument` node in your area-specific `etc/graphql/di.xml` file.
+
+The following example adds an array-item to the `extendedConfigData` array within the construct of the `StoreConfigDataProvider`.
+
+```xml
+<?xml version="1.0" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+	<type name="Magento\StoreGraphQl\Model\Resolver\Store\StoreConfigDataProvider">
+		<arguments xsi:type="array">
+			<argument name="extendedConfigData">
+				<item name="section_group_field" xsi:type="string">section/group/field</item>
+			</argument>
+		</arguments>
+	</type>
+</config>
+```
+
+You must also extend the type `storeConfig` within in the `etc/schema.graphqls` file, as shown below:
+
+```graphql
+type StoreConfig {
+    section_group_field : String  @doc(description: "Extended Config Data - section/group/field")
+}
+```
+
```

Comment:
Create a H2 called Output attributes. Bump the "Supported" headings down to H3.

Reply:
done